sopel/module.py

```diff
@@ -41,3 +41,21 @@
     url,
     VOICE,
 )
+
+
+def intent(*intent_list):
+    """Decorate a callable to trigger on intent messages.
+
+    :param str intent_list: one or more intent(s) on which to trigger (really,
+                            the only useful value is ``ACTION``)
+
+    .. versionadded:: 5.2.0
+
+    .. deprecated:: 7.1
+
+    .. seealso::
+
+        This will be removed in Sopel 9, as the intent feature in IRCv3 spec is
+        long dead. You can use :func:`@ctcp <sopel.plugin.ctcp>` instead.
```

Comment:
This should perhaps also be `.. important::`, and adapt some of the wording changes I applied to the `Trigger.ctcp` property's docstring if you like. I've taken a stab at that below.

```suggestion
    .. important::

        This will be removed in Sopel 9, as the IRCv3 intent specification is
        long dead. You can use :func:`@ctcp <sopel.plugin.ctcp>` instead.
```